src/scss/_mixins.scss

```diff
@@ -89,14 +89,16 @@
 		}
 	}
 
-	.o-footer__matrix-title,
 	.o-footer__matrix-link__copy {
 		overflow: hidden;
-		white-space: nowrap;
 		text-overflow: ellipsis;
```

Comment:
comment: I think this property is now no longer required because the text will now wrap instead

Reply:
The design for this component does not wrap text, it hides text behind ellipsis (for better or worse). As an inline element  `text-overflow` is having no affect here, but we can restore `white-space: nowrap;` and set`display: block;` to get the same affect as in the design.

Reply:
@notlee Still I get the same overflow after setting `display: block;`

![Screen Shot 2021-07-13 at 11 48 32 AM](https://user-images.githubusercontent.com/9023404/125431107-5e902ef2-b5d7-4088-96a7-6b6ce062735c.png)